Document/0x03-Overview.md

```diff
@@ -1,75 +1,86 @@
 
-## Why Does the World Need a Mobile Application Security Testing Guide?
+>The second paragraph that used to be here is kinda repeated below in Local Data Storage, so I removed the duplicate content and just hinted at it, keeping the explanation down below. 
 
-Every new technology introduces new security risks, and mobile computing is no different. Even though modern mobile operating systems like iOS and Android are arguably more secure by design compared to traditional Desktop operating systems, there's still a lot of things that can go wrong when security is not considered during the mobile app development process. Data storage, inter-app communication, proper usage of cryptographic APIs and secure network communication are only some of the aspects that require careful consideration.
+## Key Areas in Mobile Application Security (AppSec)
 
-Security concerns in the mobile app space differ from traditional desktop software in some important ways. Firstly, while not many people opt to carry a desktop tower around in their pocket, doing this with a mobile device is decidedly more common. As a consequence, mobile devices are more readily lost and stolen, so adversaries are more likely to get physical access to a device and access any of the data stored.
+Many mobile application penetration testing tools have a background in network and web app penetration (app pen) testing, a quality that is valuable for mobile app testing. Almost every mobile app talks to a backend service, and those services are prone to the same kinds of attacks we are familiar with in web apps on desktop machines. Mobile apps differ in that there is a smaller attack surface and therefore more security against injection (attackers supply input to a program) and similar attacks. Instead, we must prioritize data protection on the device and the network to increase mobile security. 
 
-## Key Areas in Mobile AppSec
+Let's discuss the key areas in mobile application security.
 
-Many mobile app pen testers have a background in network and web app penetration testing, and a lot of their knowledge is useful in mobile app testing. Practically every mobile app talks to some kind of backend service, and those services are prone to the same kinds of attacks we all know and love. On the mobile app side however, there is only little attack surface for injection attacks and similar attacks. Here, the main focus shifts to data protection both on the device itself and on the network. The following are some of the key areas in mobile app security.
+>Maybe it's a good idea to number each security area below to keep it tied into this introductory sentence above? Maybe not, let me know your thoughts.
```

Comment:
No, I think the section headers make sure of that. I.e. these sections with three hashes `###` all belong to the `## Key Areas in Mobile Application Security (AppSec)` section with two hashes.